source/Makefile

```diff
@@ -8,6 +8,12 @@ LIBS = -lm -lpthread
 OPTS = -Ofast -march=native -std=c++14 -pedantic -m64 ${INCLUDES}
 HONG = -D__LCAO
 HONG += -D__ELPA
+ifeq ($(OPENMP), ON)
+    HONG += -D_OPENMP
```

Comment:
```suggestion
```

Reply:
So you mean we should leave HONG '_OPENMP' alone to let the compiler decide? I don't think compiler could add HONE to the compilation script. In Makefile, all HONGs are added manually.

Reply:
> So you mean we should leave HONG '_OPENMP' alone to let the compiler decide? I don't think compiler could add HONE to the compilation script. In Makefile, all HONGs are added manually.

Not in Makefile script. The compiler will add the flag if OpenMP feature is utilized. 
Ref: https://www.openmp.org/spec-html/5.0/openmpse10.html

Reply:
OK I understand that. I would fix here.